src/test/java/de/meggsimum/w3w/What3WordsTest.java

```diff
@@ -69,10 +103,29 @@ public void testPositionToWords() {
 	}
 
 	/**
+	 * Tests the position with a different language-> words API wrapper
+	 */
+	public void testPositionToFrenchWords() {
+		What3Words w3w = new What3Words(API_KEY);
+		double[] coords = {49.422636, 8.320833};
+		String[] words;
+		try {
+			words = w3w.positionToWords(coords, "fr");
+			assertEquals(3, words.length);
+			assertEquals("besacier", words[0]);
+			assertEquals("trimer", words[1]);
+			assertEquals("effectuer", words[2]);
+		} catch (Exception e) {
+			assertTrue(false);
+		}
+	}
+
+	/**
 	 * Tests the position -> words API wrapper after changing the language
 	 */
 	public void testChangeLang() {
 		What3Words w3w = new What3Words(API_KEY);
+		String defaultLanguage = w3w.getLanguage();
```

Comment:
Hi, yes it does not have a real sens but in fact, it tests the getLanguage method. I should a real test with it.